packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -16,9 +16,7 @@ MeetingsUtil.extractDestination = (destination, type) => {
     if (!(destination && destination.url)) {
       throw new ParameterError('You cannot create a meeting by locus without a locus.url defined');
     }
-    const temp = destination.url.split('/');
-
-    dest = temp[temp.length - 1];
+    dest = destination.url;
```

Comment:
?

Reply:
Is this a breaking change?

Reply:
when we pass locus url as a destination we strip the locusId and use it get meetingInfo. We want the whole locus Url to get the proper meetingInfo